apps/atlas/src/app/search/details/field/field-details.component.html

```diff
@@ -11,7 +11,7 @@ <h3 class="title is-3">
       <div>
         <p class="heading">
           <a models4insightDetailsNavigationItem="#related-attributes">

```

Comment:
Looks like we hard coded this but it seems like we could also use the translations here. Maybe something to change now while we are touching this part anyways? Same for the other hard coded names in the other files.

See the example below of how to apply translation here.

```suggestion
          <a models4insightDetailsNavigationItem="#related-attributes" translate>
```

The value below would need to change to the key in the translations file. Looks like `attributes` would maybe be appropriate here.